pcap_set_fanout.3pcap

```diff
@@ -0,0 +1,48 @@
+.\" Copyright (c) 1994, 1996, 1997
+.\"	The Regents of the University of California.  All rights reserved.
+.\"
+.\" Redistribution and use in source and binary forms, with or without
+.\" modification, are permitted provided that: (1) source code distributions
+.\" retain the above copyright notice and this paragraph in its entirety, (2)
+.\" distributions including binary code include the above copyright notice and
+.\" this paragraph in its entirety in the documentation or other materials
+.\" provided with the distribution, and (3) all advertising materials mentioning
+.\" features or use of this software display the following acknowledgement:
+.\" ``This product includes software developed by the University of California,
+.\" Lawrence Berkeley Laboratory and its contributors.'' Neither the name of
+.\" the University nor the names of its contributors may be used to endorse
+.\" or promote products derived from this software without specific prior
+.\" written permission.
+.\" THIS SOFTWARE IS PROVIDED ``AS IS'' AND WITHOUT ANY EXPRESS OR IMPLIED
+.\" WARRANTIES, INCLUDING, WITHOUT LIMITATION, THE IMPLIED WARRANTIES OF
+.\" MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE.
+.\"
+.TH PCAP_SET_FANTOUT 3PCAP "10 February 2018"
+.SH NAME
+pcap_set_fanout \- set fanout group for load balancing
+among processes
+.SH SYNOPSIS
+.nf
+.ft B
+#include <linux/if_packet.h>
+#include <pcap/pcap.h>
+.LP
+.ft B
+int pcap_set_fanout(pcap_t *p, int flags, int group_id);
+.ft
+.fi
+.SH DESCRIPTION
```

Comment:
This section should use the same language that the pcap_set_protocol(3pcap) page does to indicate that 1) this is Linux-specific, and the function isn't even *available* on other platforms and 2) it only affects network interfaces, not other devices.